easy-to-use-api/klips-chart-api/src/constants/index.ts

```diff
@@ -11,9 +11,14 @@ export const pathNameConfig: PathNameConfig = {
 };
 
 export const processURL = window.location.href.indexOf('localhost') > -1 ?
-'https://klips-dev.terrestris.de/processes/location-info-time-rasterstats/execution' :
+  'https://klips-dev.terrestris.de/processes/location-info-time-rasterstats/execution' :
```

Comment:
Non-blocking, but you should consider keeping the hosts in a configuration file, as they may change on another server.